standard_rules/brute_force_by_ip.py

```diff
@@ -1,3 +1,12 @@
+'''
+Searching logs in GSuite:
+
+SELECT *
+FROM panther_logs.public.gsuite_reports
+WHERE id:applicationName = 'login'
+AND events[0]:name = 'login_failure' LIMIT 10;
+'''
```

Comment:
Saved/scheduled query CLI support will be ready in 1.16 I think, then we can start a dedicated directory/pack of cool queries. Excited for that.